unit-tests/functional/alt-IR/test-enable-while-streaming.cpp

```diff
@@ -0,0 +1,32 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+
+#include "../../tests.h"
+#include "../utilities.h"
```

Comment:
Maybe it should be called `functional.h`? That was it's clear where it is? What do you think?

Reply:
done